components/proof-of-yield/DripperGraph.tsx

```diff
@@ -0,0 +1,190 @@
+import React, { forwardRef } from "react";
+import { GradientButton } from "../../components";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  Tooltip,
+  TimeScale,
+  BarElement,
+  ChartData,
+  ChartOptions,
+} from "chart.js";
+import { poyBarOptions as barOptions } from "../../chart-configs";
+import { createTooltip, tailwindConfig } from "../../utils";
+import { twMerge } from "tailwind-merge";
+import { Typography } from "@originprotocol/origin-storybook";
+import { Bar } from "react-chartjs-2";
+import { commify } from "ethers/lib/utils";
+
+ChartJS.register(CategoryScale, TimeScale, BarElement, Tooltip);
+
+const colors = tailwindConfig.theme.colors;
+
+interface DripperGraphProps {
+  setTime?: boolean;
+  graphId: number;
+  title: string;
+  extraOptions?: Partial<ChartOptions<"bar">>;
+  extraData?: {
+    title: string;
+    value: string;
+  }[];
+  className?: string;
+  bgClassName?: string;
+}
+
+enum ChartTime {
+  SEVEN_DAY = 7,
+  THIRTY_DAY = 30,
+  ONE_YEAR = 365,
+}
+
+const dataX = [];
+const dataY = [];
+
+for (let i = 0; i < 30; i++) {
+  dataX.push(Date.now() + i * 24 * 60 * 60 * 1000);
+  dataY.push(((Math.random() * 100000) % 1500) + 2500);
```

Comment:
Is this just stubbed out data for now?

Reply:
Yeah this is a component for the dripper page (See https://www.ousd.com/proof-of-yield/dripper) that is not being implemented in this PR.